.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: c
+sudo: required
+dist: trusty
+matrix:
```

Comment:
By using keyword "proposal", you mean that for now, the travis works with ubuntu, but not with apex distribution.
right ?